docs/Contributing.md

```diff
@@ -0,0 +1,75 @@
+Branches
+========
+
+The `master` branch contains the latest code that is not necessarily released.
+At any time, the code on the `master` branch should be suitable for releasing.
+
+Developer Workflow
+==================
+
+Creating a Pull Request
+-----------------------
+
+When you start work on a patch, make sure you're assigned to the bug. If you
+don't have privileges you can request to be assigned by commenting on the bug or
+asking in [irc](https://wiki.mozilla.org/Loop#Communication_Channels).
+
+* When working on a patch it should be in a fork of the main repository.
+  * It is advisable to use a separate well named branch for each patch.
+  * Keep the branch/patch rebased on the latest `master` (especially before making
+a pull request).
+* Whilst working on your patch, make sure it passes `make test`, especially before
+creating a pull request.
+* Once you've finished a bug, commit the patch with a meaningful commit message
+which describes why it is being changed, for example:
+  * `Bug 123456 – Loop's RoomList view requires user profile data passed in when
+it doesn't need to.`
+* If you have multiple commits you should squash them down into one before
+creating a pull request, unless there's a reason for separating them, for example:
+  * With commits as one patch the patch would be a significant size and make it
+hard for review.
+  * When it makes sense to separate commits logically, e.g. “Implement a feature
+change”, “Remove code associated with previous version of feature”
+* Go to your repository in github, and create a pull request to the `master`
```

Comment:
Might be overly explicit, but one needs to push the branch to the github loop fork to see the pull request option.